### PR TITLE
Xwayland fix and wayland stack version bumps

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
You need to reset the revision on wayland and wayland-protocols and bump the revision on xorg-server

File: bootstrap.d/dev-libs.yml [hunk 1]
```diff
@@ -1443,8 +1443,8 @@ packages:
     source:
       subdir: 'ports'
       git: 'https://github.com/wayland-project/wayland-protocols.git'
-      tag: '1.24'
-      version: '1.24'
+      tag: '1.26'
+      version: '1.26'
     tools_required:
       - system-gcc
       - wayland-scanner
```
Comment (ArsenArsen) [inline bootstrap.d/dev-libs.yml:1450]: this shouldn't be revision 2 after the bump

**Review by ArsenArsen — CHANGES_REQUESTED** [review]

**Comment by no92** [comment]
Addressed the comments raised above and updated `xorg-server` and split out `xwayland`.

**Comment by no92** [comment]
Reordered and re-signed the commits.

**Comment by no92** [comment]
Included a bump to weston and included weston's new dependency on seatd in order to resolve Xwayland issues.

File: bootstrap.d/dev-libs.yml [hunk 2]
```diff
@@ -1497,6 +1496,7 @@ packages:
       - pango
       - glib
       - libjpeg-turbo
+      - seatd
     revision: 3
```
Comment (Dennisbonke) [inline bootstrap.d/dev-libs.yml:1500]: The revision needs to be removed as we're upgrading versions.

File: bootstrap.d/dev-libs.yml [hunk 3]
```diff
@@ -1497,6 +1496,7 @@ packages:
       - pango
       - glib
       - libjpeg-turbo
+      - seatd
```
Comment (Dennisbonke) [inline bootstrap.d/dev-libs.yml:1499]: We should replace the dependency on `xorg-server` with `xwayland`.

File: bootstrap.d/x11-base.yml [hunk 4]
```diff
@@ -152,3 +149,62 @@ packages:
       - args: ['ninja', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: xwayland
```
Comment (Dennisbonke) [inline bootstrap.d/x11-base.yml:153]: Missing metadata, feel free to add it if you feel like maintaining this part of the Xorg stack. If you rather have me maintain it, let me know and I'll take it.

File: bootstrap.d/x11-libs.yml [hunk 5]
```diff
@@ -2027,3 +2027,28 @@ packages:
       - args: ['make', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: libxcvt
```
Comment (Dennisbonke) [inline bootstrap.d/x11-libs.yml:2031]: Missing metadata, feel free to add it if you feel like maintaining this part of the Xorg stack. If you rather have me maintain it, let me know and I'll take it.
Also, you're missing the `labels: [aarch64]` thing, as it stands this build _will_ fail on xbbs for aarch64.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Some small things that need to be fixed but overall this PR looks nice.

File: bootstrap.d/x11-base.yml [hunk 6]
```diff
@@ -152,3 +149,69 @@ packages:
...
+    labels: [aarch64]
+    architecture: '@OPTION:arch@'
+    source:
+      subdir: ports
+      git: 'https://gitlab.freedesktop.org/xorg/xserver.git'
+      tag: 'xwayland-22.1.3'
+      version: '22.1.3'
+    metadata:
```
Comment (Dennisbonke) [inline bootstrap.d/x11-base.yml:161]: Wrong location, other ports put metadata above the source declaration. For consistency's sake we should do the same here.

File: bootstrap.d/x11-libs.yml [hunk 7]
```diff
@@ -2027,3 +2027,36 @@ packages:
...
+    labels: [aarch64]
+    architecture: '@OPTION:arch@'
+    source:
+      subdir: ports
+      git: 'https://gitlab.freedesktop.org/xorg/lib/libxcvt.git'
+      tag: 'libxcvt-0.1.2'
+      version: '0.1.2'
+    metadata:
```
Comment (Dennisbonke) [inline bootstrap.d/x11-libs.yml:2039]: Wrong location, other ports put metadata above the source declaration. For consistency's sake we should do the same here.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Two minor nitpicks, other than that looks good.

**Comment by no92** [comment]
Rebased, fixed up the fixups and addressed the comments above.

**Review by Dennisbonke — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**